src/store/reducers/packagesReducer.js

```diff
@@ -93,7 +102,9 @@ const updatePackagesReducer = (state, { payload, fromAction }) => {
 
 /* Action types that cause a pending flag */
 const fetchableTypes = [
-  epics.actionTypes.fetchPackages
+  epics.actionTypes.fetchPackages,
+  epics.actionTypes.createPackage,
```

Comment:
I don't think these new ones are considered fetchable. Did you talk with Stephen?

Reply:
Yes I talked with Stephen. If you notice this flyout doesn't do API call from itself. Instead it delegates that work to epics and hence this change to track errors.

This design is a lot cleaner than the previous ones.